### PR TITLE
Deactivate campaigns where terminate == True

### PR DESCRIPTION
**Pull request by zeantsoi**
:eyeglasses: @dwick 

Per [reddit-public #5095](https://github.com/reddit/reddit-public/pull/5095).

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -396,6 +397,7 @@ def update_flight(link, campaign, triggered_by=None):
         'IsActive': (not campaign_needs_approval and
```
Comment (dwick) [inline reddit_adzerk/adzerkpromote.py]: if you have time can you make this entire thing into a testable function

**Comment by dwick** [comment]
can you add `is_terminated` to `additional_data` in the events:
https://github.com/reddit/reddit-plugin-adzerk/blob/master/reddit_adzerk/adzerkpromote.py#L561-L566

💅

**Comment by zeantsoi** [comment]
:haircut:

File: reddit_adzerk/adzerkpromote.py [hunk 2]
```diff
@@ -347,6 +347,22 @@ def update_advertiser(author, triggered_by=None):
...
+    is_inactive = (
+        needs_approval or
+        is_paused or
+        needs_payment or
+        is_terminated or
+        is_deleted or
+        is_overdelivered
+    )
+
+    if is_inactive:
```
Comment (dwick) [inline reddit_adzerk/adzerkpromote.py]: `return not is_inactive` will do here

**Comment by dwick** [comment]
:fish:

**Comment by zeantsoi** [comment]
4f5f7c7e25eb5756517f7e53f639efca4543ee8b
24709b6221c43ef61699b2055a83108dc2e739db

**Pull request closed without merging**